test/Microsoft.AspNetCore.SignalR.Tests.Utils/ServerLogScope.cs

```diff
@@ -49,6 +50,11 @@ private void ServerFixtureOnServerLogged(LogRecord logRecord)
             // https://github.com/aspnet/Logging/issues/810
```

Comment:
This issue has been fixed. Move the the `logger = _serverLoggers.GetOrAdd(...)` call out of the lock and remove the comment.

Reply:
Keeping it in the lock as that's what is preventing a race where we would otherwise get an ODE.

But removed the comment :D